src/SuperResponsiveTable.js

```diff
@@ -1,12 +1,32 @@
 import React from 'react'
 import { Provider, Consumer } from './tableContext'
+import { injectBaseStyles } from './baseStyleUtils'
+
+const classes = ({ bem, others }) => {
+  const bemArray = Array.isArray(bem) ? bem : [bem]
+  const othersArray = Array.isArray(others) ? others : [others]
+
+  const allClasses = [
+    ...bemArray.map(name => `super-responsive-table__${name}`),
```

Comment:
I'm using `BEM`-style selectors to minimize the specificity of our base styles.

Reply:
BEM-style is better